controllers/base.go

```diff
@@ -52,7 +52,7 @@ func (c *ApiController) IsGlobalAdmin() bool {
 		return false
 	}
 
-	return user.Owner == "built-in" || user.IsGlobalAdmin
+	return user.Owner == "built-in" || user.IsAdmin
```

Comment:
IsAdmin is org admin, not global admin

Reply:
IsGlobalAdmin is useless for all admin users(including "built-in" and other orgs), but IsAdmin is useful for orgs except "built-in",I use this function to determine if the current user is an administrator